pages/k8s/1.29/components.md

```diff
@@ -0,0 +1,199 @@
+| [azure-cloud-provider](https://charmhub.io/azure-cloud-provider) | Charm which enables Azure out-of-tree integrations for Charmed Kubernetes | [source](https://github.com/charmed-kubernetes/charm-azure-cloud-provider.git) | [docs](https://charmhub.io/azure-cloud-provider/docs) | [bugs](https://bugs.launchpad.net/charm-azure-cloud-provider) |
+| [azure-integrator](https://charmhub.io/azure-integrator) | Charm which enables Azure integrations for Juju | [source](https://github.com/juju-solutions/charm-azure-integrator.git) | [docs](https://charmhub.io/azure-integrator/docs) | [bugs](https://bugs.launchpad.net/charm-azure-integrator) |
+| [gcp-cloud-provider](https://charmhub.io/gcp-cloud-provider) | Charm which enables GCP out-of-tree integrations for Charmed Kubernetes | [source](https://github.com/charmed-kubernetes/charm-gcp-cloud-provider.git) | [docs](https://charmhub.io/gcp-cloud-provider/docs) | [bugs](https://bugs.launchpad.net/charm-gcp-cloud-provider) |
+| [gcp-integrator](https://charmhub.io/gcp-integrator) | Charm which enables Google GCP integrations for Juju | [source](https://github.com/juju-solutions/charm-gcp-integrator.git) | [docs](https://charmhub.io/gcp-integrator/docs) | [bugs](https://bugs.launchpad.net/charm-gcp-integrator) |
+| [gcp-k8s-storage](https://charmhub.io/gcp-k8s-storage) | Charm which enables out-of-tree GCP storage for Charmed Kubernetes. | [source](https://github.com/charmed-kubernetes/gcp-k8s-storage.git) | [docs](https://charmhub.io/gcp-k8s-storage/docs) | [bugs](https://bugs.launchpad.net/charm-gcp-k8s-storage) |
+| [openstack-integrator](https://charmhub.io/openstack-integrator) | Charm which enables OpenStack integrations for Juju | [source](https://github.com/juju-solutions/charm-openstack-integrator.git) | [docs](https://charmhub.io/openstack-integrator/docs) | [bugs](https://bugs.launchpad.net/charm-openstack-integrator) |
+| [vsphere-cloud-provider](https://charmhub.io/vsphere-cloud-provider) | Charm which enables vSphere out of tree provider for Juju | [source](https://github.com/charmed-kubernetes/vsphere-cloud-provider.git) | [docs](https://charmhub.io/vsphere-cloud-provider/docs) | [bugs](https://bugs.launchpad.net/charm-vsphere-cloud-provider) |
+| [vsphere-integrator](https://charmhub.io/vsphere-integrator) | Charm which enables vSphere integrations for Juju | [source](https://github.com/juju-solutions/charm-vsphere-integrator.git) | [docs](https://charmhub.io/vsphere-integrator/docs) | [bugs](https://bugs.launchpad.net/charm-vsphere-integrator) |
+
+
+## Additional charms
+
+These charms are also maintained and supported by the Charmed Kubernetes team
+to extend the capabilities of your Kubernetes cluster.
+
+|charm | summary | source | docs | bugs |
+|-|-|-|-|-|
+| [aws-iam](https://charmhub.io/aws-iam) |  | [source](https://github.com/charmed-kubernetes/charm-aws-iam.git) | [docs](https://charmhub.io/aws-iam/docs)  | [bugs](https://bugs.launchpad.net/charm-aws-iam) |
```

Comment:
I think the template that generates all these docs links was written before they changed how they appear on CH. The '/docs' isn't needed

Reply:
ack that it may not be needed, but like the prev comment, this comes from the `charm-support-matrix.inc` in the jenkins repo and that's semi-frozen for 1.29 while sqa does their thing.  we can revisit when auto-generating the 1.29+ck1 rel notes.